<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Insertcomputer</class>
 <widget class="QDialog" name="Insertcomputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Insert a computer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_namecomputer">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_namecomputer"/>
   </item>
   <item>
    <widget class="QLabel" name="label_yearbuilt">
     <property name="text">
      <string>Year built</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_yearbuilt"/>
   </item>
   <item>
    <widget class="QLabel" name="label_type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_type"/>
   </item>
   <item>
    <widget class="QLabel" name="label_wasitbuilt">
     <property name="text">
      <string>Was it built?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_wasitbuilt"/>
   </item>
   <item>
    <widget class="QPushButton" name="submit_computer">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
